setup.cfg

```diff
@@ -55,6 +55,93 @@ test =
     hypothesis>=5.5.3
     pytest>=6.0
     pytest-xdist>=1.31
+# optional extras for recommended dependencies
+# see: doc/source/getting_started/install.rst
+performance =
+    bottleneck>=1.3.2
+    numba>=0.53.0
+    numexpr>=2.7.1
+computation =
+    numba>=0.53.0
+    scipy >=1.7.1
+    xarray>=0.19.0
+aws =
+    boto3>=1.22.7
+    s3fs>=0.4.0
+gcp =
+    gcsfs>=2021.05.0
+    pandas-gbq>=0.15.0
+excel =
+    odfpy>=1.4.1
+    openpyxl>=3.0.7
+    pyxlsb>=1.0.8
+    xlrd>=2.0.1
+    xlwt>=1.3.0
+    xlsxwriter>=1.4.3
+parquet =
+    pyarrow>=1.0.1
+feather =
+    pyarrow>=1.0.1
+hdf5 =
+    blosc>=1.20.1
+    tables>=3.6.1
+spss =
+    pyreadstat>=1.1.2
+postgresql =
+    SQLAlchemy>=1.4.16
+    psycopg2>=2.8.6
+mysql =
+    SQLAlchemy>=1.4.16
+    pymysql>=1.0.2
+sql-other =
+    SQLAlchemy>=1.4.16
+html =
+    beautifulsoup4>=4.9.3
+    html5lib>=1.1
+    lxml>=4.6.3
+plot =
+    matplotlib>=3.3.2
+output_formatting =
+    jinja2>=3.0.0
+    tabulate>=0.8.9
+compression =
+    brotlipy>=0.7.0
+    python-snappy>=0.6.0
+    zstandard>=0.15.2
+# `all` supersets all the above options.
+# Should be maintained as the complete set of pandas optional dependencies
+all =
```

Comment:
I think this section is still missing some of the dependencies in `ci/deps/actions-38-minimum_verions.yaml`

Reply:
Which ones?  Not sure what I should modify here.

- fastparquet is excluded as strictly redundant and inferior to pyarrow as per [Tom's comment](https://github.com/pandas-dev/pandas/issues/39164#issuecomment-1178989749)
- fsspec is missing because it is a explicit s3fs, gcsfs dependency that it is better to let them handle
- pytables is a typo in [actions-38-minimum_verions.yaml](https://github.com/pandas-dev/pandas/blob/main/ci/deps/actions-38-minimum_versions.yaml), which should be [tables](https://pypi.org/project/tables/) to match the package name on PyPi.  (`tables` is included in `all`)
- testing-specific libraries are not present as presumably out of scope for general-use pandas install that `all` facilitates.  (I'm happy for anyone who wants to get  under the hood to do `pandas[test,all]`).  Let me know if you disagree.

Reply:
I think for a "consistent" user experience, `all` should mean all dependencies listed in `doc/source/getting_started/install.rst` even if there are some redundant or unneeded dependencies